view/frontend/layout/catalog_product_view.xml

```diff
@@ -7,7 +7,7 @@
 -->
 <page xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance" xsi:noNamespaceSchemaLocation="urn:magento:framework:View/Layout/etc/page_configuration.xsd">
     <body>
-        <referenceBlock name="breadcrumbs" template="Opengento_ProductBreadcrumbs::product/breadcrumbs.phtml">
+        <referenceBlock name="breadcrumbs" template="Opengento_ProductBreadcrumbs::product/breadcrumbs.phtml" aria-label="Breadcrumb" role="navigation">
```

Comment:
Those attributes are not available in the layout instructions. Could you revert this line change please?

Reply:
You are right, this makes no sense there. 
Will change it asap.